src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -840,17 +840,17 @@ class DetailsCard extends React.Component {
                       }
                     </FieldRow>
                     <FieldRow label={msg.memory()} id={`${idPrefix}-memory`}>
-                      { !isFullEdit && `${round(memorySize)} ${memoryUnit}` }
+                      { !isFullEdit && `${round(memorySize / (1024 ** 3), 2)} ${'GiB'}` }
```

Comment:
I meant not to hard-code `GiB`, but use function `userFormatOfBytes` to calculate most preferable unit with precision.

Reply:
didn't know this existed!